pkg/sink/stdiosink/colors.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/lucasb-eyer/go-colorful"
 )
 
```

Comment:
This needs to be turned into an argument to a function somewhere. We don't want to introduce package level variable.